pyproject.toml

```diff
@@ -11,7 +11,7 @@ requires-python = ">=3.8"
 dependencies = [
     "Jinja2 >=2.9",
     "contourpy >=1",
-    "numpy >=1.11.3",
+    "numpy >=1.20.0",
```

Comment:
This needs consideration. 1.11.3 is pretty old, maybe we want to bump the min version, but not as far as 1.20 I don't think

Reply:
In any case, a version bump should land in 3.1 but I intend to put the import fix in a 3.0.2 very soon since there is another bugfix with no WAR that needs to be released. So let's just drop this change, and make a separate issue about bumping the min version.

Reply:
`contourpy` has a runtime requirement of `numpy>=1.16` which we pick up indirectly. We could make the same change here and it wouldn't affect any end users.

Reply:
I was thinking 1.16 as well, let's do it in a separate PR for 3.1 tho

Reply:
closing the circle: https://github.com/bokeh/bokeh/pull/12602